lib/charms/opensearch/v0/opensearch_backups.py

```diff
@@ -305,6 +305,21 @@ def _restore(self, backup_id: int) -> Dict[str, Any]:
 
         return output["snapshot"]
 
+    def is_idle_or_not_set(self) -> bool:
```

Comment:
I believe `is_idle` is enough. I would understand it as "not in the middle of something" - regardless of the real state of the idleness

Reply:
@Mehdi-Bendriss not according to this other review: https://github.com/canonical/opensearch-operator/pull/267#discussion_r1584427948

And indeed, this name makes it more complete.